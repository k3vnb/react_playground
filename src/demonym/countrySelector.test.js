import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import CountrySelector from './countrySelector'

describe('<CountrySelector /> component', () => {
    it('renders upon load', () => {
        expect(toJson(shallow(<CountrySelector />))).toMatchSnapshot()
    })
})
