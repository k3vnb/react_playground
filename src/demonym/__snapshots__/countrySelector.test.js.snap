// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CountrySelector /> component renders upon load 1`] = `
<div
  className="country_selector"
>
  <form>
    <label
      htmlFor="country"
    >
      Select a country:
    </label>
    <select
      id="country"
      name="country"
    >
      <option
        value="None"
      >
        Select one...
      </option>
    </select>
  </form>
</div>
`;
